compass/ocean/__init__.py

```diff
@@ -34,3 +35,4 @@ def __init__(self):
         self.add_test_group(Soma(mpas_core=self))
         self.add_test_group(SphereTransport(mpas_core=self))
         self.add_test_group(Ziso(mpas_core=self))
+        self.add_test_group(SphericalHarmonicTransform(mpas_core=self))
```

Comment:
Please keep these alphabetical (same with the imports above, though that's less critical).